internal/controller/snapshot/snapshot_adapter.go

```diff
@@ -522,9 +522,14 @@ func (a *Adapter) HandlePipelineCreationError(err error, integrationTestScenario
 		return controller.RequeueWithError(itsErr)
 	}
 
+	if strings.Contains(err.Error(), "admission webhook") {
```

Comment:
Maybe we could expand this condition to also require `denied the request` to be present in the error message?

Reply:
Searched for elegant way how to check for multiple strings. None of them were elegant.